api/src/test/java/org/pmiops/workbench/api/ConceptsControllerTest.java

```diff
@@ -233,7 +233,7 @@
   private static User currentUser;
 
   @TestConfiguration
-  @Import({WorkspaceServiceImpl.class, WorkspaceMapper.class})
+  @Import({WorkspaceServiceImpl.class, WorkspaceConversionUtils.class})
```

Comment:
This no longer needs to be auto wired right?

Reply:
Exactly. I have changes for that locally.